apps/andi/lib/andi/input_schemas/datasets.ex

```diff
@@ -274,6 +333,10 @@ defmodule Andi.InputSchemas.Datasets do
   defp extract_owner_id(_, %{ownerId: ownerId}), do: ownerId
   defp extract_owner_id(_, _), do: nil
 
+  defp extract_organization_id(nil, %{organization_id: organization_id}), do: organization_id
```

Comment:
Is this one necessary if the next one catches nil with the `_` and doesn't do anything functionally different?

Reply:
That's a good question- This code was directly modeled on the `extract_owner_id` function above it. We will do some testing to see if all three variations of the function are needed.

Reply:
Good catch, @ksmith-accenture. Fixed!